spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfigDefaults.java

```diff
@@ -0,0 +1,97 @@
+package org.springframework.cloud.netflix.ribbon;
+
+import java.util.concurrent.TimeUnit;
+
+import com.netflix.client.config.DefaultClientConfigImpl;
+
+/**
+ * This is shadowing the defaults defined in {@link DefaultClientConfigImpl}, this way the
+ * rest of the code can refer to this class over {@link DefaultClientConfigImpl} and any
+ * changes to the defaults carry over.
+ */
+
+public class RibbonClientConfigDefaults {
```

Comment:
There is still the problem that if new properties are added we wont know.  This solves the problem if the values change.

Reply:
sorry, totally missed out your review comment @ryanjbaxter . Yes, it is true that we will not catch other upstream changes to `DefaultClientConfigImpl` . Considering that we there are other plans to move away from Archaius 1 and too many files touched by this change  I am good to close this PR without merging if you are okay with it @ryanjbaxter

Reply:
Yeah I think that makes sense